### ISSUE TITLE
Display number of Results fetched

### ISSUE THREAD

**Issue by rupav** [fossasia/query-server#314]
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**

There is no display of number of results fetched.
**Expected behavior:**

Diplaying the number of results fetched.
**Steps to reproduce:**

**Comment by rupav** [comment]
@bhaveshAn @cclauss  your views on it?

**Comment by cclauss** [comment]
Can we please get more detail on the output format that you propose?
* Today: print('Google parsed: ' + str(urls))

Do you propose?:
* print('Google ({}): {}'.format(len(urls), urls))
* print('{} results from Google: {}'.format(len(urls), urls))
* print('{} results from Google:\n{}'.format(len(urls), urls))
* print('{} results from Google:\n{}'.format(len(urls), ‘\n’.join(str(url) for url in urls)))
* print('{} results from Google:\n{}'.format(len(urls), json.dumps(urls, indent=4)))
* other?

**Comment by rupav** [comment]
@cclauss I am having a problem about the codebase, pinged you on gitter. Please see. Thanks.

**Comment by cclauss** [comment]
Is you issue that when you run locally, you see nothing printed?

**Comment by rupav** [comment]
@cclauss I opened this issue for displaying number of results on the [website](https://query-server.herokuapp.com/) itself.

**Comment by cclauss** [comment]
👌 Now I understand.

**Comment by starlord1311** [comment]
can i take up this issue? @cclauss @rupav

**Comment by rupav** [comment]
@starlord1311 yeah you can. Let's confirm if it's needed or not. @bhaveshAn @shashank-sharma please let us know if this feature is needed.

**Comment by mariobehling** [comment]
It would be nice to have that, but I am unclear if we are really able to get this data easily.
